### DISCUSSION TITLE
Can this actions-runner-controller be used in K8s versions from 1.23?

### DISCUSSION THREAD

**Discussion by LogeshGovi** [actions/actions-runner-controller#1006] [Questions] [answered]
Hi, I would like to know if dind use-cases(docker run, build ..)  work on K8s versions from 1.23 on this runner as docker-shim is deprecated.

**Comment by Moser-ss** [comment]
Hey, I test the actions-runner-controller in a GKE cluster, that uses containerd as ContainerRuntime. So that means don't have docker-shim and it works well.
So I have a strong opinion this controller will work in K8s version higher than 1.24

**Reply by LogeshGovi** [reply]
Thank you

**Comment by mumoshu** [comment, marked as answer]
It should just work. `docker-shim` isn't relevant in this scenario. `docker-shim` is only used to create containers contained in the pod. `dind` runs within a container of a pod and we use `docker` there, not `docker-shim` provided by K8s 😃

**Reply by LogeshGovi** [reply]
Thank you
